eras/shelley/impl/testlib/Test/Cardano/Ledger/Shelley/CDDL.hs

```diff
@@ -164,30 +153,57 @@ move_instantaneous_rewards_cert =
 
 move_instantaneous_reward :: Rule
 move_instantaneous_reward =
-  "move_instantaneous_reward"
-    =:= arr [a (int 0 / int 1), a $ mp [0 <+ asKey stake_credential ==> coin]]
+  comment
+    [str|The first field determines where the funds are drawn from.
+        |  0 denotes the reserves,
+        |  1 denotes the treasury.
+        |If the second field is a map, funds are moved to stake credentials.
+        |Otherwise, the funds are given to the other accounting pot.
+        |NOTE:
+        |  This has been safely backported to Shelley from Alonzo.
+        |]
+    $ "move_instantaneous_reward"
+      =:= arr
+        [ a (int 0 / int 1)
+        , a (smp [0 <+ asKey stake_credential ==> delta_coin] / coin)
```

Comment:
Yes, this change was done in `alonzo`. And we already solved this `shelley` onwards, in the implementation, but the CDDL for `shelley`, `allegra` and `mary` didn't reflect the change. Since the CDDL didn't enforce it but on the implementation level, it is already handled, I back-ported the pending change as part of this PR.